docs/source/models/cause.rst

```diff
@@ -275,14 +281,125 @@ Birth Prevalence
+		population of interest for 6 months, they contribute 0.5 person-years.
+		Together, these individuals contribute a total of 2.5 person-years.
+
+	**At-risk population:** the at-risk population is defined as the 
+	population that *does not* have the condition of interest; in other words, 
+	the susceptible population that is at risk of developing the condition. 
+	Notably, the number of individuals in this population will change overtime 
+	as the following events occur:
+
+	   - Members of the at-risk population develop the condition and are no longer susceptible
+	   - Members of the at-risk population die and are no longer susceptible 
+	   - Individuals are born or age into the at-risk population and become susceptible
+	   - Individuals with the condition recover from the condition and re-enter the at-risk population as susceptible (in the case of conditions with remission)
+
+Because the denominator for cumulative incidence is person-time in the at-risk 
+population, it can represent the probability of a new case of the condition 
+occuring in an individual without the condition in a given time frame. 
+Therefore, it can be used to represent the probability that a simulant will 
+transition from a susceptible to infected cause model state in a given 
+timestep.
+
+  For instance, consider an example in which the global cumulative incidence
+  of injuries in 2017 was 6,800 cases per 100,000 person-years, or 0.068 cases
+  per person-year. In this example, 6,800 new injuries occurred among 100,000 
+  person-years of observation among the non-injured population.
+
+  Now, consider a cause model with a susceptible (not injured) state and an 
+  infected (injured) state with a simulation timestep of 1 year. In this case, 
+  the probability that a simulant will transition from the susceptible to 
+  infected state within a single timestep (i.e. the transition probability) 
+  would be represented as 0.068.
+
+  Notably, in order to represent the transition probability for a single 
+  simulant within a single timestep, the cumulative incidence value needs to 
+  be scaled so that the person-time denominator is equal to the simulation 
+  timestep. Therefore, if the timestep of the cause model considered above 
+  were six months instead of one year, the transition probability would be 
+  0.034 (0.034 cases per 0.5 person-years). 
+
+Cumulative incidence contrasts with other measures of incidence that may have 
+different denominators. For instance, incidence rate may refer to new cases of 
+a given condition that occur in a specified timeframe (e.g. 10 cases per year) 
+or may refer to new cases per person-time of the general population rather 
+than the at-risk population. Incidence measures other than cumulative with 
+person-time of the at-risk population as the denominator are not appropriate 
+for incidence data sources for cause model transition probabilities.
```

Comment:
Again, change cumulative incidence to (average) incidence rate.

Ideally, I would like some context about when these different measures of rates are appropriate.

For example, why does GBD use total population person-time rather than at-risk person-time when reporting incidence?

Reply:
Interesting... I am not exactly sure, but my guess is that incidence/total_pop is a more relevant measure for policy discussions than incidence/susceptible_pop. Incidence/total_pop allows for a quick measure of "what percent of my country's population will develop the disease this year". With incidence/susceptible_pop, a quick calculation (including condition prevalence) would be necessary to figure out that number.

Do you think this would be worth discussing in this section?

Reply:
Hmm, yes, I think it would be worth putting in a sentence or two explaining what GBD does and why, especially given this comment from below:

> Also, for context: as discussed with @collijk, our transformation is simply un-doing a transformation that GBD made to get incidence in the total population from an estimate of incidence in the susceptible population.

It looks like the todo mentions this, so perhaps we can add something in a future version.